source/create.js

```diff
@@ -27,16 +26,23 @@ const create = defaults => {
 		defaults.handler = (options, next) => next(options);
 	}
 
+	if (defaults.handler.constructor.name !== 'AsyncPromise') {
```

Comment:
This is not a reliable check. What are you trying to do here?

Reply:
Nevermind. I'll fix it :P